data-model/SDP-data-model-templates/codicological-unit_sdp-template.json

```diff
@@ -204,15 +225,33 @@
     "viewing_hint": "CV",
     "range": "???"
   },
-  "contributor": [
-    "array of strings (CV)"
-  ],
-  "related_objects": [
-    {
-      "relationship_type": "CV",
-      "object": "ARK",
-      "position": "int",
-      "layer": "int"
-    }
-  ]
+  "related_objects": {
+    "has_content": [
```

Comment:
is 'has_content' too broad? Would/could illustrations eventually get their own records? Coming from the other direction, if we don't give paratexts their own records, we really are only interested in describing with this relationship the major contents of the ms